nixos/tests/all-tests.nix

```diff
@@ -604,9 +604,10 @@ in {
   sway = handleTest ./sway.nix {};
   switchTest = handleTest ./switch-test.nix {};
   sympa = handleTest ./sympa.nix {};
-  syncthing = handleTest ./syncthing.nix {};
-  syncthing-init = handleTest ./syncthing-init.nix {};
-  syncthing-relay = handleTest ./syncthing-relay.nix {};
+  syncthing = handleTest ./syncthing/syncthing.nix {};
+  syncthing-init = handleTest ./syncthing/syncthing-init.nix {};
+  syncthing-relay = handleTest ./syncthing/syncthing-relay.nix {};
+  syncthing-declarative = handleTest ./syncthing/syncthing-declarative.nix {};
```

Comment:
```suggestion
  syncthing = handleTest ./syncthing {};
  syncthing-init = handleTest ./syncthing/init.nix {};
  syncthing-relay = handleTest ./syncthing/relay.nix {};
  syncthing-declarative = handleTest ./syncthing/declarative.nix {};
```
maybe, not sure

Reply:
it would be nice, now that I think of it I would keep `syncthing-relay = handleTest ./syncthing-relay.nix {};` 
@SuperSandro2000 do you think it would be good to move syncthing relay in the top directory for tests again?  syncthing-relay is a software on its own.